### ISSUE TITLE
`StringList.create` conflict with RequiousFracto

### ISSUE THREAD

**Issue by Krutoy242** [friendlyhj/ZenUtils#81]
With mod [RequiousFracto](https://www.curseforge.com/minecraft/mc-mods/requious-frakto) installed, `StringList.create` returns different values based on variable declaration method.

test.zs:

```less
#reloadable
import mods.zenutils.StringList;

print('~'~StringList.create(['one', 'two', 'three']).get(1));

val list = ['one', 'two', 'three'] as string[];
print('~'~StringList.create(list).get(1));
```

crafttweaker.log:
```ruby
[SERVER_STARTED][SERVER][INFO] ~two
[SERVER_STARTED][SERVER][INFO] ~requious.util.Parameter$Constant@17342467
```

**Comment by friendlyhj** [comment]
:concern:
If you are using it in tab completion, you can use `IGetTabCompletion.fixedValues`
About other use cases... I should make some hacky.

**Comment by Krutoy242** [comment]
Yes, im using it [here](https://github.com/Krutoy242/Enigmatica2Expert-Extended/blob/3d4518f91d341faf10199393e435369959d312f0/scripts/commands/build.zs#L42). I will try to use `fixedValues`.

**Comment by friendlyhj** [comment]
btw, you reinvented another wheel
https://github.com/friendlyhj/ZenUtils/wiki/ZenCommandTree

**Comment by Krutoy242** [comment]
Yes, i probably just forgot it exist...
